schemas/content/product-attributes.schema.json

```diff
@@ -0,0 +1,193 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/content/product-attributes",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Product Attributes",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "description": "",
+  "definitions": {
+    "attributes": {
+      "properties": {
+        "schema:color": {
+          "title": "Color",
+          "$ref": "#/definitions/attribute",
+          "description": "The color of the product or varinat."
+        },
+        "xdm:size": {
+          "title": "Size",
+          "$ref": "#/definitions/attribute",
+          "description": "The size of the product or varinat."
+        },
+        "xdm:style": {
+          "title": "Style",
+          "$ref": "#/definitions/attribute",
+          "description": "The sytle description of the product or varinat."
+        },
+        "xdm:type": {
+          "title": "Type",
+          "$ref": "#/definitions/attribute",
+          "description": "The type of the product or varinat."
+        },
+        "xdm:model": {
+          "title": "Model",
+          "$ref": "#/definitions/attribute",
+          "description": "The model of the product or varinat."
+        },
+        "xdm:features": {
+          "title": "Features",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/attribute"
+          },
+          "description": "Descriptive feature list of the product or varinat."
+        },
+        "xdm:productLine": {
+          "title": "Product Line",
+          "$ref": "#/definitions/attribute",
+          "description": "The product line of the product or varinat."
+        },
+        "schema:weight": {
```

Comment:
I see no reason to use third party schemas for these simple definitions, It may even be wrong unless the `#/definitions/attribute` is from schema.org too. And considering it includes `xdm:` properties I think it does not.
@trieloff What do you think, I recommend converting all these `schema:*` attributes to first class `xdm:*` attributes.

Reply:
We should use `schema:*` attributes whenever we can, as they make the meaning unambiguous. XDM is derived from Schema.org, so there is no need to re-define these properties.